pkgs/servers/web-apps/netbox/generic.nix

```
@@ -92,6 +92,7 @@
       '';
 
       passthru = {
+        inherit (python3) pythonVersion;
```

```suggestion
        python = python3;
```
probably better to just passthru python